### PR TITLE
Advising dash styles

### PR DESCRIPTION
**Pull request by jwfung**
Here's how it looks:

image

### CODE REVIEW

**Review by rbiz4 — APPROVED** [review]

File: static/js/redux/ui/advisor_panel.jsx [hunk 1]
```diff
@@ -1,91 +1,91 @@
...
-    semester_name: PropTypes.string,
-    semester_year: PropTypes.string,
-
-  }),
-  displayed_advisees: PropTypes.arrayOf(PropTypes.object),
-  displayAdvisee: PropTypes.func.isRequired,
-};
-
-export default AdvisorPanel;
+/*
```
Comment (JJamesWWang) [inline static/js/redux/ui/advisor_panel.jsx:1]: Looks like whitespace got messed up again

Comment (JJamesWWang) [reply static/js/redux/ui/advisor_panel.jsx:1]: Should set your default line break to LF instead of CRLF

**Review by JJamesWWang — CHANGES_REQUESTED** [review]
Line breaks

File: static/js/redux/ui/advisor_panel.jsx (hunk 1 above)
Comment (jwfung) [reply static/js/redux/ui/advisor_panel.jsx]: fixed it!

**Review by JJamesWWang — APPROVED** [review]

**Comment by Boccca2014** [comment]
Can you make the test bold and larger? And can you also to make it have a gray background similar to the MVP?

**Comment by Boccca2014** [comment]
It now looks like this, let me know if any of you think this should be updated further:
Screen Shot 2021-04-28 at 5 20 25 PM

**Comment by miaboloix** [comment]
Screen Shot 2021-04-28 at 6 25 26 PM

**Comment by Boccca2014** [comment]
I think Jeanie has some changes that she didn't commit because I made changes but moved them to #272

**Review by Boccca2014 — APPROVED** [review]

**Pull request merged**